### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by TomaszPartyka0990**
I started with basic domain classes, from now i can move on with some first services for saving and displaing those beings (connected with Thymeleaf views)

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+spring.datasource.url=jdbc:h2:mem:testdb
```
Comment (kchrusciel) [inline src/main/resources/application.properties:1]: Maybe you should use profiles, or H2 is your production database?

File: src/main/java/pl/sda/partyka/service/RoleService.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import pl.sda.partyka.domain.Role;
+import pl.sda.partyka.repository.RoleRepository;
+
+@Service
+@RequiredArgsConstructor
+public class RoleService {
+
+    private final RoleRepository RoleRepo;
```
Comment (kchrusciel) [inline src/main/java/pl/sda/partyka/service/RoleService.java:12]: JavaCode conventions, the field should be camelCased

File: src/main/java/pl/sda/partyka/DataInitializer.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.stereotype.Component;
+import pl.sda.partyka.domain.Role;
+import pl.sda.partyka.service.RoleService;
+
+@Component
+@RequiredArgsConstructor
+public class DataInitializer implements ApplicationRunner {
```
Comment (kchrusciel) [inline src/main/java/pl/sda/partyka/DataInitializer.java:12]: Are you sure if this is a part of the production code? Maybe you should use database initializer https://stackoverflow.com/a/38047021/4414894

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-security</artifactId>
+        </dependency>
+        -->
+
+    </dependencies>
+
+    <build>
+        <plugins>
+            <!-- Buduje wykonywalny JAR -->
```
Comment (kchrusciel) [inline pom.xml:80]: This should be in english :D

File: src/main/resources/application.properties (hunk 1 above)
Comment (TomaszPartyka0990) [reply src/main/resources/application.properties:1]: I used H2 on this phase just to see if relations are created as I imagined after app build. Later on i will change it for MySQL or create 2 different profiles for dev and production

**Review by kchrusciel — APPROVED** [review]
Looks good, waiting for the next stuff ;)

**Pull request merged**